dev/RadioButtons/TestUI/RadioButtonsPage.xaml

```diff
@@ -36,6 +36,10 @@
                             <x:Double>1.0</x:Double>
                             <x:Double>2.0</x:Double>
                         </controls:RadioButtons>
+
+                        <!-- Checking that the DefaultRadioButtonStyle can be accessed as resource -->
+                        <RadioButton Style="{StaticResource DefaultRadioButtonStyle}" Content="I am a RadioButton with DefaultRadioButtonStyle set ... meep morp" />
```

Comment:
>... meep morp [](start = 147, length = 13)

While fun, I think we should avoid the Steven Universe references. Sorry :(

Reply:
I am deeply sorry about the reference (which I didn't even intend to be a reference to anything). I have removed it now.

Reply:
Absolutely don't worry about it, I sincerely doubt it would ever be an issue, but just in case I'd says better to leave it out.